pkg/addon/policyframework/manifests/managedclusterchart/templates/role.yaml

```diff
@@ -17,6 +17,12 @@ metadata:
     heritage: {{ .Release.Service }}
     addon.open-cluster-management.io/hosted-manifest-location: hosting
 rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
```

Comment:
I thought since CRDs are cluster-wide resources, this would need to be in a ClusterRole? Am I missing something? https://kubernetes.io/docs/reference/access-authn-authz/rbac/#clusterrole-example